types/sd-notify/index.d.ts

```diff
@@ -0,0 +1,52 @@
+// Type definitions for sd-notify 2.8
+// Project: https://github.com/systemd/node-sd-notify#readme
+// Definitions by: Jan Beseda <https://github.com/Cheprer>
+//                 Rory Bradford <https://github.com/roryrjb>
```

Comment:
```suggestion
//                 Rory Bradford <https://github.com/roryrjb>
```
the ACK of this user is required, if you want to have this handle in the source.
DT is sending notifications to owners, so ACK would be required here, thanks!

Reply:
:+1: from me